src/services/validation.js

```diff
@@ -1,9 +1,11 @@
 import firebaseAPI from "@/services/API/checkUser.js";
 import { sendPlio } from "@/services/sendPlio";
 
-//this function is invoked only for the check of SRN's.
-export async function validateSRN(userID, validateCount, isSingleEntryOnly, redirectID, doesUserExist){
+import {sendSQSMessage} from "@/services/API/sqs";
 
+//this function is invoked only for the check of SRN's.
+export async function validateSRN(userID, validateCount, isSingleEntryOnly, redirectID, doesUserExist, purpose, purposeParams, redirectTo){
+    var authType="SRN"
     var invalidLoginMessage = ""
     //checks the basic conditions of SRN: 
     //      - starts with only '1'
```

Comment:
I don't think this is true btw. The SRN can start with 2 also. Just check in the sheet, I think I saw one that starts with a `2`.